lib/rgeo/geographic/projected_feature_methods.rb

```diff
@@ -19,7 +19,7 @@ def projection
       end
 
       def envelope
-        factory.unproject(projection.envelope)
+        factory.unproject(projection.unsafe_envelope)
```

Comment:
What's the reasoning for us having to use the unsafe versions of methods in the projected features?

Reply:
If we call the safe versions, we have some issues:

- first we end up running `check_validity` multiple times,
- second if one wants to call for instance `proj_factory.unsafe_area`, it will itself call `projection.area`, which checks for validity.